### ISSUE TITLE
Unexpected word token: "generated" when parsing sql

### ISSUE THREAD

**Issue by dschinkel** [oguimbal/pg-mem#9]
Not sure how to fix this during parsing.  I took at `pg_dump` of my prod database, removed the roles, etc. and only left table creation and constraints in the dump, but I get this error

```
ALTER TABLE public.city ALTER COLUMN city_id ADD GENERATED
                                                                                                                                     ^
Unexpected word token: "generated". I did not expect any more input
```

**for this part of the script:**
```js
ALTER TABLE public.city ALTER COLUMN city_id ADD GENERATED ALWAYS AS IDENTITY (
  SEQUENCE NAME public.city_city_id_seq
  START WITH 0
  INCREMENT BY 1
  MINVALUE 0
  NO MAXVALUE
  CACHE 1
);

```

**Also got an error on this**
```
ADD CONSTRAINT city_id_fk FOREIGN KEY (city_id) REFERENCES public.
                                                                       ^
Unexpected dot token: ".". Instead, I was expecting to see one of the following:

- A "lparen" token
```

**for:**

```js
ALTER TABLE ONLY public.location
    ADD CONSTRAINT city_id_fk FOREIGN KEY (city_id) REFERENCES public.city(city_id) MATCH FULL;
```

**Comment by oguimbal** [comment]
Released with pg-mem@1.4.5

**Comment by dschinkel** [comment]
Really appreciate this man!  I'll try it out tonight!!
